cpp/src/parquet/encoding.cc

```diff
@@ -17,6 +17,7 @@
 
 #include "parquet/encoding.h"
 
+#include <emmintrin.h>
```

Comment:
This should probably be done in arrow/util/sse_util.h

Reply:
+1

Reply:
Done